packages/docs/src/css/custom.css

```
@@ -31,6 +30,18 @@ h1 {
   --ifm-color-primary-lightest: var(--ifm-color-primary);
   --ifm-code-font-size: 95%;
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.1);
+
+  --ifm-font-family-base:
+    'Inter Variable', -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto',
```

Stylelint (17.6.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)